website/src/common/Macrobench.tsx

```diff
@@ -138,73 +138,73 @@ export default function Macrobench(props: MacrobenchProps) {
             left={fixed(data.diff.Left.Result.threads, 2)}
             right={fixed(data.diff.Right.Result.threads, 2)}
             diffMetric={fixed(data.diff.Diff.threads, 2)}
-          />
+          /> */}
```

Comment:
Why

Reply:
the response returned by api  does not have specific values that correspond to these .This is the reason they are commented out.